### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by NikitaNevv**
https://NikitaNevv.github.io/layout_html-form/
https://NikitaNevv.github.io/layout_html-form/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,7 +11,220 @@
...
+            name="dateOfBirth"
+            id="birth"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form1">
+          <label for="policyAgr">
+          I accept the term of the agreement
+          </label>
```
Comment (SanyaBratashchuk) [inline src/index.html:76]:
```suggestion
          <label for="policyAgr">
            I accept the term of the agreement
          </label>
```

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,220 @@
...
+          <input
+          type="checkbox"
+          id="policyAgr"
+          name="termOfAgreement"
+          >
```
Comment (SanyaBratashchuk) [inline src/index.html:81]:
```suggestion
          <input
            type="checkbox"
            id="policyAgr"
            name="termOfAgreement"
          >
```

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,220 @@
...
+        <div class="margin-bottom">
+          <label for="comm">
+            Comments:
+          </label>
+          <textarea
+            name="comments"
+            id="comm"
+            autocomplete="off"
+            >
+          </textarea>
```
Comment (SanyaBratashchuk) [inline src/index.html:208]:
```suggestion
          <textarea
            name="comments"
            id="comm"
            autocomplete="off"
          ></textarea>
```

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 4]
```diff
@@ -11,7 +11,219 @@
...
-    <h1>HTML Form</h1>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+    >
+      <fieldset class="fieldset">
+        <legend>
+          Personal information
+        </legend>
+        <div class="form1">
```
Comment (alexandra-protyanova) [inline src/index.html:22]: divs are redundant in this task

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]

File: src/index.html [hunk 5]
```diff
@@ -11,7 +11,213 @@
...
+            id="howOld"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form">
```
Comment (alexandra-protyanova) [inline src/index.html:59]: I meant remove all divs

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
Do not close your pr, it doesn't have a lot of sense

**Comment by NikitaNevv** [comment]
> Do not close your pr, it doesn't have a lot of sense

thanks, i thought it wouldnt work without it

File: src/index.html [hunk 6]
```diff
@@ -11,7 +11,215 @@
...
+            id="comm"
+            autocomplete="off"
+         ></textarea>
+        </label>
+
+        <label for="reccomend">Would you recommend us?</label>
+        <select
+          name="reccomendUs"
+          id="reccomend"
+          autocomplete="off">
```
Comment (DarkMistyRoom) [inline src/index.html:212]:
```suggestion
        <select
          name="reccomendUs"
          id="reccomend"
          autocomplete="off"
        >
```

File: src/index.html [hunk 7]
```diff
@@ -11,7 +11,215 @@
...
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>
+          An interesting fact about you!
+        </legend>
+        
+        <label class="label">
```
Comment (DarkMistyRoom) [inline src/index.html:122]: not really good idea to use a common label here because when you click on the question, yes became selected

**Review by DarkMistyRoom — CHANGES_REQUESTED** [review]

**Pull request closed without merging**